### PR TITLE
add first knockout binds!

### LINKED ISSUE #13: update battle window
- [x] Add currency bar
- [x] Add battle stat bar

### PR DESCRIPTION
**Pull request by dennism1997**
closes #13

### CODE REVIEW

File: src/styles/main.css [hunk 1]
```diff
@@ -3,8 +3,9 @@
     src: url(../assets/fonts/firered.ttf);
 }
 
-body{
-    background: url('../assets/images/background.png') center;
+body {
+    background: url('../assets/images/background.png') top;
+    background-size: cover;
 }
```
Comment (Ishadijcks) [inline src/styles/main.css]: Does this mean the background always stretches to fullscreen?

File: src/scripts/PlayerView.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
+/// <reference path="Player.ts"/>
+
```
Comment (Ishadijcks) [inline src/scripts/PlayerView.ts]: Do you have to bind using a class? That seems kind of redundant?

File: src/styles/main.css (hunk 1 above)
Comment (dennism1997) [inline src/styles/main.css]: Yeah it scales it to the viewport

File: src/scripts/PlayerView.ts (hunk 2 above)
Comment (dennism1997) [inline src/scripts/PlayerView.ts]: Not really sure. It doesn't seem to update the value too :/

Comment (dennism1997) [inline src/scripts/PlayerView.ts]: I just made this class to keep it separated

File: src/scripts/Player.ts [hunk 3]
```diff
@@ -3,17 +3,25 @@
...
     // TODO Eh not a big fan of this name.
     private static _notCaughtBallSelection: GameConstants.Pokeball = GameConstants.Pokeball.Masterball;
     private static _alreadyCaughtBallSelection: GameConstants.Pokeball = GameConstants.Pokeball.Pokeball;
 
+    public static clickAttackObservable : KnockoutComputed<number> = ko.computed(function () {
+        return Player.calculateClickAttack()
+    });
+
+    public static pokemonAttackObservable : KnockoutComputed<number> = ko.computed(function () {
+        return Player.calculatePokemonAttack(null, null);
```
Comment (Ishadijcks) [inline src/scripts/Player.ts]: Should be GameConstants.PokemonType.None

File: src/scripts/Player.ts [hunk 4]
```diff
@@ -3,17 +3,25 @@
  * All player variables need to be saved.
  */
 class Player {
-    private static _money: number;
+    private static _money: KnockoutObservable<number> = ko.observable(0);
```
Comment (Ishadijcks) [inline src/scripts/Player.ts]: The 0 here refers to the value of money right?

Comment (dennism1997) [inline src/scripts/Player.ts]: yeah, needed a starting value

Comment (Ishadijcks) [inline src/scripts/Player.ts]: Alright, just double checking it wasn't some ko value.

**Pull request merged**